auron-build.sh

```diff
@@ -24,6 +24,7 @@ print_help() {
     echo "Build Auron project with specified Maven profiles"
     echo
     echo "Options:"
+    echo "  --docker <true|false>    Build in Docker (default: false)"
```

Comment:
Currently, seems only support to build with centos7 Dockerfile.

Maybe we need to provide more details in the usage.

And in the future, we may support more kinds of OS and customized Dockerfile.